### PR TITLE
OKTA-656585 - Add example to Users API

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** As per this i[ssue](https://github.com/okta/okta-developer-docs/issues/2612), added an example to the Users API
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-656585](https://oktainc.atlassian.net/browse/OKTA-656585)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 1]
```diff
@@ -2679,16 +2679,37 @@ curl -v -X POST \
...
 ```
 
 ##### Response example
 
-
 ```json
 {}
 ```
 
+##### Request example (Convert a user to an Okta User)
```
Comment (mingchili-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md:2691]: "an user"?

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 2]
```diff
@@ -2679,16 +2679,37 @@ curl -v -X POST \
...
+
+To convert a user to an Okta user, pass an empty value as the `provider` in the [Provider object](#provider-object). The `sendEmail`
+parameter must be false or omitted for this type of conversion.
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/users/{userId}/lifecycle/reset_password?provider=&sendEmail=false"
```
Comment (mingchili-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md:2701]: I think you can just remove the `provider=` part

Comment (mingchili-okta) [reply packages/@okta/vuepress-site/docs/reference/api/users/index.md:2701]: Hmm seems `sendEmail` isn't mandatory either.

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md (hunk 1 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/reference/api/users/index.md:2691]: No, "a user" is correct in this scenario. English is a funny language.

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md (hunk 2 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/reference/api/users/index.md:2701]: Yes, tested and you're correct. I'll make that update. It's basically the default call, but I think the example might help.

**Review by mingchili-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 3]
```diff
@@ -2621,7 +2621,8 @@ Generates a one-time token (OTT) that can be used to reset a user's password.  T
 This operation will transition the user to the status of `RECOVERY` and the user will not be able to login or initiate a forgot password flow until they complete the reset flow.
 
 This operation provides an option to delete all the user' sessions.  However, if the request is made in the context of a session owned by the specified user, that session isn't cleared.
->**Note:** You can also use this API to convert a user with the Okta Credential Provider to a use a Federated Provider. After this conversion, the user cannot directly sign in with password. The second example demonstrates this usage.
+
+>**Note:** You can also use this API to convert a user with the Okta Credential Provider to a use a Federated Provider. After this conversion, the user cannot directly sign in with password. The second example demonstrates this usage. To return a federated user back to an Okta user, use the default API call. See the final example.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md:2625]:
```suggestion
>**Note:** You can also use this API to convert a user with the Okta Credential Provider to a use a Federated Provider. After this conversion, the user can't directly sign in with a password. The second example demonstrates this usage. To return a federated user back to an Okta user, use the default API call. See the final example.
```

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 4]
```diff
@@ -4263,7 +4281,7 @@ Specifies the authentication provider that validates the user's password credent
 
 ### Links object
 
-Specifies link relations (see [Web Linking](http://tools.ietf.org/html/rfc8288) available for the current status of a user.  The Links object is used for dynamic discovery of related resources, lifecycle operations, and credential operations.  The Links object is read-only.
+The Links object specifies link relations (see [Web Linking](http://tools.ietf.org/html/rfc8288) available for the current status of a user).  The Links object is used for dynamic discovery of related resources, lifecycle operations, and credential operations.  The Links object is read-only.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md:4284]:
```suggestion
The Links object specifies link relations (see [Web Linking](http://tools.ietf.org/html/rfc8288) available for the current status of a user). The Links object is used for dynamic discovery of related resources, lifecycle operations, and credential operations. The Links object is read-only.
```

**Review by barbaravo-okta — APPROVED** [review]
Looks good!

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/users/index.md](https://github.com/okta/okta-developer-docs/blob/762c90eb90d8c2f1b00312b2fe0498983ee863c7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fusers%2Findex.md) | 69 | 492 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/da720681-c29a-4dec-b181-5b2c87d233ae) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.36956db7-2cd0-44cc-b7bc-1f67bd7a599f?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 5]
```diff
@@ -2621,7 +2621,8 @@ Generates a one-time token (OTT) that can be used to reset a user's password.  T
 This operation will transition the user to the status of `RECOVERY` and the user will not be able to login or initiate a forgot password flow until they complete the reset flow.
 
 This operation provides an option to delete all the user' sessions.  However, if the request is made in the context of a session owned by the specified user, that session isn't cleared.
->**Note:** You can also use this API to convert a user with the Okta Credential Provider to a use a Federated Provider. After this conversion, the user cannot directly sign in with password. The second example demonstrates this usage.
+
+>**Note:** You can also use this API to convert a user with the Okta Credential Provider to a use a Federated Provider. After this conversion, the user can't directly sign in with a password. The second example demonstrates this usage. To return a federated user back to an Okta user, use the default API call. See the final example.
```
Comment (timhowie-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md:2625]:
```suggestion
>**Note:** You can also use this API to convert a user with the Okta Credential Provider to a use a Federated Provider. After this conversion, the user can't directly sign in with a password. The second example demonstrates this usage. To convert a federated user back to an Okta user, use the default API call. See the final example.
```

**Review by timhowie-okta — APPROVED** [review]
lgtm

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/users/index.md](https://github.com/okta/okta-developer-docs/blob/5f36465ffce455bae789f1cb634edc5ac06ff1f8/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fusers%2Findex.md) | 69 | 490 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a2d4dd3a-2698-4deb-979b-5d7cc862098e) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.dbd75793-be87-475b-85e9-ce895d152b15?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Pull request merged**